index.js

```diff
@@ -1,43 +1,58 @@
-'use strict';
+'use strict'
```

Comment:
You seem to have rewritten this file without trailing newlines? Was this the result of an auto-formatter or `prettier` run? (If so, we should check in the config). Otherwise, is it possible to keep the newlines for consistency?

Reply:
I have a linter installed on my Atom which uses js-standard. If you want, we can have it back but, I noticed that I was doing stuff like lines without `;` without noticing and then you had them with `;`. That's why I manually formatted the code to the rules of https://standardjs.com/

Reply:
@reyalpsirc please respect the coding conventions you see in the files you edit when doing a PR.  Thanks. :) (And wishing your feature was already merged!)